src/hotspot/share/gc/shared/threadLocalAllocBuffer.cpp

```diff
@@ -473,3 +474,11 @@ size_t ThreadLocalAllocBuffer::end_reserve() {
   size_t reserve_size = Universe::heap()->tlab_alloc_reserve();
   return MAX2(reserve_size, (size_t)_reserve_for_allocation_prefetch);
 }
+
+const HeapWord* ThreadLocalAllocBuffer::start_relaxed() const {
```

Comment:
Why do we need the "relaxed" terminology? By default accessors have no memory ordering properties.

Reply:
I'm not sure what you are asking, but Atomic::load implements the "relaxed" semantics equivalent to using MO_RELAXED in the Access API.